circleci/circleci-docs
# [CIRCLE-15698] Webpack fixes

# Description
from the ticket: 
Currently in the circleci-docs repo if you make any changes to the portion of that site's JS that is compiled via webpack (i.e. src-js/app.js), you need to remember to commit the compiled output (jekyll/assets/js/dist/app.min.js) or else your changes will not be reflected in the production build. We should fix our build so that this is not required.

While we're at this, it probably makes sense to stop minifying the JS compiled by webpack in development.

# Reasons
Jira ticket: https://circleci.atlassian.net/browse/CIRCLE-15698

# How to Test
- Make a change to one of the webpack-compiled files
- Run the webpack production build: `yarn run webpack-prod`
- Build the production bundle: `bundle exec jekyll build --config jekyll/_config.yml,jekyll/_config_production.yml --source jekyll --destination jekyll/_site/docs/ 2>&1 | tee $JOB_RESULTS_PATH/build-results.txt`
- Check that the changes you made to the JS files are reflected in the production bundle at `/jekyll/_site/docs/assets/dist/app-xxxxxxxxx.js`
